extensions/-happen3-/additionalroots.js

```diff
@@ -0,0 +1,110 @@
+// Name: Additional Roots
+// ID: additionalroots
+// Description: A collection of new additional roots for your enjoyment.
+// By: -happen3-
+// Original: -happen3-
```

Comment:
No need to put yourself at the original, as you already have yourself down as the creator.

Reply:
I don't know why I did put that.